mlflow/utils/async_logging/run_batch.py

```diff
@@ -9,27 +9,28 @@
 class RunBatch:
     def __init__(
         self,
-        run_id: str,
-        params: List[Param],
-        tags: List[RunTag],
-        metrics: List[Metric],
-        completion_event: threading.Event,
-    ) -> None:
+        run_id: Optional[str],
```

Comment:
```suggestion
        run_id: str,
```
nit: let's update type hint